Scripts/build.swift

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env swift
+
+import Foundation
+
+// Usage: build.swift platforms
+
+func execute(commandPath: String, arguments: [String]) throws {
+    let task = Process()
+    task.launchPath = commandPath
+    task.arguments = arguments
+    print("Launching command: \(commandPath) \(arguments.joined(separator: " "))")
+    task.launch()
+    task.waitUntilExit()
+    guard task.terminationStatus == 0 else {
+        throw TaskError.code(task.terminationStatus)
+    }
+}
+
+enum TaskError: Error {
+    case code(Int32)
+}
+
+enum Platform: String, CaseIterable, CustomStringConvertible {
+    case iOS_13
```

Comment:
We aren't using these older versions in CI, but they can be useful for running tests on older simulator OS versions locally.